collectors/auth0/test/auth0_test.js

```diff
@@ -154,6 +154,46 @@ describe('Unit Tests', function () {
                 done();
             });
         });
+
+        it('log format when type is null or undefined', function (done) {
+            let ctx = {
+                invokedFunctionArn: auth0Mock.FUNCTION_ARN,
+                fail: function (error) {
+                    assert.fail(error);
+                    done();
+                },
+                succeed: function () {
+                    done();
+                }
+            };
+
+            Auth0Collector.load().then(function (creds) {
+                var collector = new Auth0Collector(ctx, creds);
+                let fmt = collector.pawsFormatLog(auth0Mock.AUTH0_LOG_EVENT.type=null);
+                assert.equal(fmt.messageTypeId, undefined);
+                done();
+            });
+        });
+
+        it('log format when date is null or undefined', function (done) {
+            let ctx = {
+                invokedFunctionArn: auth0Mock.FUNCTION_ARN,
+                fail: function (error) {
+                    assert.fail(error);
+                    done();
+                },
+                succeed: function () {
+                    done();
+                }
+            };
+
+            Auth0Collector.load().then(function (creds) {
+                var collector = new Auth0Collector(ctx, creds);
+                let fmt = collector.pawsFormatLog(auth0Mock.AUTH0_LOG_EVENT.date=null);
```

Comment:
first set the null  like auth0Mock.AUTH0_LOG_EVENT.date=null.
then pass  AUTH0_LOG_EVENT in function . collector.pawsFormatLog(auth0Mock.AUTH0_LOG_EVENT)